_posts/2016-07-04-imam-week1.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "CrowdCensus Team's Data Exploration"
+date:   2016-07-04
+author: Imam and Myeong
+---
+
+The goal of CrowdSensing Census project in DSSG 2016 is to develop a reliable and general model that can complement census data by making use of other real-time data such as OpenStreetMap data or Cellphone Detail Records (CDR) to determine the social economic conditions in a particular region. This work is meaningful for allowing data freshness and cost efficiency compared to census data, since government-initiated socio-economic statistics are expensive and often outdated (10-year term). 
+
+Specifically, a prediction model would be developed from available datasets targeting the city of Milano and the province of Trento, Italy. We target these cities as the Big Data Challenge initiative made it possible to open Telecom Italia data (CDR) to the public <a href="http://www.nature.com/articles/sdata201555" target="_blank">(Barlacchi, et al., 2015)</a>. For the ground truths, we are trying to make use of the census data that include, but are not limited to, the statistics about illiteracy, school degree, foreigners, rented dwelling, unemployment and work force. We first conducted Principle Component Analysis (PCA) for the non-monetary variables about citizen well-beings. The PCA was used to see whether the variables represent a single dimension of socio-economic status.
+
+<img src="../assets/images/milan-deprivation.png" align="left" style="margin: 0 10px 10px 0"> 
+
+
+Currently, the first component of the PCA result represents only about 40% of the variables, so it may not be a strong indicator of a single measure for the socio-economic level. Despite the weakness, we tried to use the measure to see the relationship with the CDR data (i.e., aggregated call-in, call-out, sms-in, sms-out, and internet usage) for the data exploration. As expected, the regressions showed that there were no significant relationships between cellphone activities and quasi (?) socio-economic levels. This was expected result, because cellphone activities vary a lot depending on time and day. To further the analyses, we have extracted time- and day-specific cellphone activity values and will conduct more granular analyses this week. 
```

Comment:
"represents only about 40% of the variance "=> "accounts for only about 40% of the variance in these variables"

"Despite the weakness" => "Despite this weakness"

"for the data exploration" => "as an exploratory data analysis"

I would take out the "As expected" in the sentence about the lack of correlation. You state that it was expected, and why, in the next sentence.